packages/superset-ui-color/src/SequentialScheme.ts

```diff
@@ -1,7 +1,7 @@
 import { scaleLinear } from 'd3-scale';
-import ColorScheme from './ColorScheme';
+import ColorScheme, { ColorSchemeConfig } from './ColorScheme';
 
-function range(count) {
+function range(count: number): number[] {
```

Comment:
can `number[]` be inferred here? (I know our style guide said rely on inference when possible, this also seems fine to me)